sdk/attestation/azure-security-attestation/test/ut/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ add_executable (
      attestation_collateral.cpp 
      attestation_metadata.cpp
      policycertmgmt_test.cpp
-     tpmattestation_test.cpp)
+     tpmattestation_test.cpp "batchtest.cpp")
```

Comment:
why the quotes?

Reply:
Because Visual Studio is really annoying w.r.t. cmake and insists on adding quotes? And messing up line formatting?

I'll fix this if I ever make another update to this PR.